src/components/faceVerification/screens/IntroScreen.jsx

```diff
@@ -57,6 +63,48 @@ const WalletDeletedPopupText = ({ styles }) => (
   </View>
 )
 
+const IntroReVerification = ({ styles, firstName, ready, onVerify, onLearnMore }) => (
+  <Wrapper>
+    <Section style={styles.topContainer} grow>
+      <View style={styles.mainContent}>
+        <Section.Title fontWeight="bold" textTransform="none" style={styles.mainTitle}>
+          {firstName && `${firstName},`}
```

Comment:
this might be empty for gooddapp

Reply:
changed to ? operator here: https://github.com/GoodDollar/GoodDAPP/pull/4303/commits/90a8e5b39654c75a404d3bf885b7082d2165e811